index.js

```diff
@@ -0,0 +1,149 @@
+// TODO list
+ // 1. TODO - fix costPerBundle calculation
+ // 2. create function for changing steps (active classes?)
+ // 5. add css for warnings instead of alerts
+ // 6. template literal for top bar?
+  // - iso numbers for caluclation on step 3
```

Comment:
You could turn each of these into issues on GitHub

Reply:
I'll start adding some of them to make a list, thanks